c_emulator/riscv_sim.cpp

```diff
@@ -439,6 +301,21 @@ static int process_args(int argc, char **argv)
       break;
     }
   }
+
+  if (!have_config) {
+    std::filesystem::path path = std::filesystem::temp_directory_path();
+    pid_t pid = getpid();
+    std::ostringstream filename;
+    filename << "default_config" << pid << ".json";
+    path = path / filename.str();
+    std::ofstream tmp(path);
+    tmp << DEFAULT_JSON;
+    tmp.close();
+
+    sail_config_set_file(path.c_str());
```

Comment:
Would be a bit nicer if we could just pass it a string directly but I guess the tempfile is fine.

Could you also add a flag like "--print-default-config" that outputs the default .json file?

Reply:
Added now